src/Roadkill.Web/Roadkill.Web.csproj

```diff
@@ -519,18 +527,23 @@
       <DependentUpon>wysiwygeditor.ts</DependentUpon>
     </Content>
     <Content Include="Assets\CSS\toastr.css" />
+    <Content Include="Plugins\SyntaxHighlighter\javascript\XRegExp.js" />
     <Content Include="Plugins\Toc\toc.js" />
     <Content Include="Plugins\Toc\toc.css" />
-    <Content Include="Roadkill.config">
+    <None Include="Roadkill.config">

```

Comment:
Need roadkill.config and connectionStrings as content, even if they're autogenerated _I think_